README.md

```diff
@@ -0,0 +1,44 @@
+# axl
+
+Python excel add-ins, made easy!
+
+# Requirements
+
+- Microsoft Excel
+- Windows 64 Bit
+- Anaconda or Miniconda
+
+# Installing
+
+In your conda environment, run `conda install -c mcg axl` to install the
+package.
+
+# Adding the Excel Plugin
+
+In the environment root folder, find the axl excel plugin at `<anaconda path>\envs\<axl env path>\axl.xlam`. 
+Add this file to your Excel addins under File -> Options -> Add-ins. The basic 
+axl functionality is now available through the Excel-Python API.
+
+# Adding Libraries
+
+To add more functions to the Excel, first ensure that the libraries you are
+going to use are installed into the conda environment are installed through pip
+or conda.
+
+axl will add a folder in your environment's Tools directory where you will list
+which functions you want included. It is located at `<anaconda path>\envs\<axl env path>\Tools\axl`.
+
+In this folder, for each library you want to use, add a file named
+`imports.<library name>`. In this file, add all of your import statements.
+axl will parse this file and add all of those imports to the Excel function
+namespace.
+
+You may need to close and open Excel for the functions to populate.
```

Comment:
We should fix this ;-)

Reply:
Fix this section?

Reply:
Oh sorry---what I meant was, we should figure out a way to reload the imports on demand.